plugins/solaris/forks

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
```

Comment:
As you said, `echo` looks enough in this case. I would like to change `<<<` to `echo`. 
But, I believe command substitution with `$()` is the feature of bash (or POSIX shell). Unfortunately, `/bin/sh` of Solaris 10 (or less) is not POSIX shell and does not support `$()`. I don't think back-quotes are good for some reason (nesting, readability...).
In my opinion, `/bin/bash` seems safer and compatible for Solaris. It should work with Solaris 8 or above.